rules/scripts/bitstreams_workspace.py

```diff
@@ -57,7 +57,7 @@
                     help="Location of the source git repo")
 parser.add_argument(
     'bitstream',
-    default='latest',
+    default='HEAD',
```

Comment:
Just a note: This change prevents the default environment from retrieving a bitstream that is not in this branch's history. As "latest" points to the most recent bitstream from a build on master, it may be for a commit that is newer than this branch's merge base.